charts/karmada/values.yaml

```diff
@@ -38,6 +38,11 @@ podDisruptionBudget: &podDisruptionBudget {}
 ## "component" means install selected components in the control-cluster
 installMode: "host"
 
+## @param installInOrder
+## ture means install karmada components in order based on their dependencies
+## false means the installation would ignore the components order and just rely on its reconciliation
+installInOrder: true
```

Comment:
At any time, we need to ensure the order of installation, this parameter seems like it should always be true, so it feels like we don't need it.

Reply:
my concern is that our method is using a `init-container`, is it possible that users can accept the restart instead of our customized init container?

Reply:
I haven't thought of scenarios where users wouldn't accept it, because it's a more user-friendly approach compared to the constant restarting method. we can look into other people's thoughts on this?

Reply:
OK, we can hang on this comment as unresolved for some time. If nobady think it is useful, we can remove this `installInOrder`. For future, if somebody truly has this need, we then add it back.